GitUI/CommandsDialogs/AboutBoxDialog/FormContributors.cs

```diff
@@ -80,7 +80,7 @@ TabPage GetNewTabPage(TextBox textBox, string caption)
                     {
                         BorderStyle = BorderStyle.None,
                         Margin = new Padding(0),
-                        Padding = new Padding(0),
+                        Padding = new(0),
```

Comment:
I like the feature because one does not need to repeat the type. But I must get used to it if the ctor has arguments.
If `Margin = new(0)` would work, I would not like it.
`Padding = new(0)` is on the borderline already because the type is not visible explicitely.

Reply:
I can revert this type of change (really reapply from scratch).